### ISSUE TITLE
transition: Accessibility map should display errors in TrRouting

### ISSUE THREAD

**Issue by tahini** [chairemobilite/transition#403]
For transit calculation, when there is an error with trRouting, the output displays a message, for example `Les données en cache pour la calculateur de chemin sont invalides. Essayez de sauvegarder les données à nouveau. Si le problème persiste, contactez les administrateurs.`

When doing accessibility map calculation, when trRouting is in that state, the same error message should be displayed. Now, it shows an isochrone for walking, as if there was not transit available instead of and error.
